internal/globalconfig/globalconfig.go

```diff
@@ -17,13 +17,15 @@ import (
 var cfg = &config{
 	analyticsRate: math.NaN(),
 	runtimeID:     uuid.New().String(),
+	headersAsTags: make(map[string]string),
 }
 
 type config struct {
 	mu            sync.RWMutex
 	analyticsRate float64
 	serviceName   string
 	runtimeID     string
+	headersAsTags map[string]string
```

Comment:
I notice that access to this isn't protected by `mu`. Do we ever expect SetHeaderTag/GetHeaderTag/etc. to be called concurrently?

Reply:
@katiehockman , when you get the chance